cps/templates/feed.xml

```diff
@@ -49,26 +49,32 @@
       </author>
     {% endif %}
     {% if entry.publishers.__len__() > 0 %}
-      <publisher>
+      <dc:publisher>
         <name>{{entry.publishers[0].name}}</name>
-      </publisher>
+      </dc:publisher>
+    {% endif %}
+    {% if entry.pubdate %}
+      <dc:issued>{{entry.pubdate}}</dc:issued>
     {% endif %}
```

Comment:
I'm not sure about this one: According to spec's example it should be 
<dc:issued>2009-10-01</dc:issued>
Calibre outputs the following:
    <updated>2013-11-30T14:17:10+00:00</updated>
    <published>2013-11-25T17:26:09+00:00</published>
    <dc:date>2012-12-05T23:00:00+00:00</dc:date>
In both cases I think the outputs needs to be formatted like for current time, and I think this was the reason I didn't do it. 
If there is no specific problem, I don't want to change it